main.s

```diff
@@ -133,12 +131,6 @@ ClearMap::
   call ClearLoop
   ret
 
-ClearVRAM::

```

Comment:
Skipping the clear on init makes sense, but I'm curious if I should keep the function around for a later time. Can't really think of a reason why at the moment, just a thought.

Reply:
For this, it's actually better to keep a general-purpose `memset` function that you call the the appropriate value.